### ISSUE TITLE
Object does not keeps the attributes submited

### ISSUE THREAD

**Issue by wbotelhos** [hzamani/active_record-acts_as#74]
When I submit:

```ruby
post :create, exam: { name: 'name', description: nil }
```

And validation fails because `description` is mandatory:

```ruby
if @exam.save
else
  render :new
end
```

The `@exam` object does not keeps the submited value.

```ruby
<%= f.text_field :name %> # nil
```

I need to manually set the `acting_as` data:

```ruby
@exam.attributes = create_params
```

Even the errors:

```ruby
@exam.acting_as.errors.each do |key, value|
  @exam.errors.add key, value
end
```

Uses `delegate` on model does not works for the attributes and I do not found a way to delegate the `errors`.

I missing something here or it is the normal behavior?

Thanks e congrats for the great gem. (:

**Comment by manuelmeurer** [comment]
Hi @wbotelhos, could you write a test or a sample app that recreates this problem? It's hard to follow without seeing your custom models and controllers.
